Mathlib/Data/Finset/Sort.lean

```diff
@@ -64,11 +64,24 @@ theorem sort_empty : sort r ∅ = [] :=
 theorem sort_singleton (a : α) : sort r {a} = [a] :=
   Multiset.sort_singleton r a
 
+theorem sort_insert [DecidableEq α] {a : α} {s : Finset α} (h₁ : ∀ b ∈ s, r a b) (h₂ : a ∉ s) :
+    sort r (insert a s) = a :: sort r s := by
```

Comment:
I wonder if a variant with `Finset.cons` would be useful, and whether you could reprove this one in terms of it

Reply:
Sure, I've added it to the PR.